src/test/java/uk/gov/justice/digital/hmpps/keyworker/services/KeyworkerAutoAllocationServiceTest.java

```diff
@@ -529,7 +532,7 @@ public void testSomeOffendersAllocatedBeforeErrorDueToNoCapacity() {
         mockUnallocatedOffenders(TEST_AGENCY_ID, getNextOffenderNo(totalOffenders));
 
         // Some available Key workers with some capacity but not enough total capacity to allocate all offenders
-        List<KeyworkerDto> someKeyworkers = mockKeyworkers(totalKeyworkers,FULLY_ALLOCATED - 2, FULLY_ALLOCATED, CAPACITY_TIER_2);
+        List<KeyworkerDto> someKeyworkers = mockKeyworkers(totalKeyworkers,FULLY_ALLOCATED - 2, FULLY_ALLOCATED, CAPACITY_TIER_1);
```

Comment:
Why changed from CAPACITY_TIER_2?

Reply:
The actual limit is still CAPACITY_TIER_2, because its now calculated as CAPACITY_TIER_1 * 9/6